packages/mdc-chips/chip/index.js

```diff
@@ -65,8 +67,19 @@ class MDCChip extends MDCComponent {
       addClass: (className) => this.root_.classList.add(className),
       removeClass: (className) => this.root_.classList.remove(className),
       hasClass: (className) => this.root_.classList.contains(className),
-      registerInteractionHandler: (evtType, handler) => this.root_.addEventListener(evtType, handler),
-      deregisterInteractionHandler: (evtType, handler) => this.root_.removeEventListener(evtType, handler),
+      addClassToLeadingIcon: (className) => {
+        if (this.leadingIcon_) {
+          this.leadingIcon_.classList.add(className);
+        }
+      },
+      removeClassFromLeadingIcon: (className) => {
+        if (this.leadingIcon_) {
+          this.leadingIcon_.classList.remove(className);
+        }
+      },
+      eventTargetHasClass: (target, className) => target.classList.contains(className),
```

Comment:
Turns out `classList` doesn't exist on SVG elements in IE 11 :( also I don't think `className` behaves the same as on other DOM elements either.

Reply:
Wrapped a div around the SVG to make IE 11 happy...the div now has the `mdc-chip__checkmark` class and the SVG has `mdc-chip__checkmark-svg`.